src/actions/effect/Effect.ts

```diff
@@ -18,7 +18,7 @@ import outline from "./outline";
 import styleTransfer from "./styleTransfer";
 import boomerang from "./simple/boomerang";
 import advancedRedEye from "./simple/advanedRedEye";
-import blackWhite from "./simple/blackWhite";
+import blackWhite from "./leveled/blackWhite";
```

Comment:
maybe just level/blackwhite

leveled has a connotation of being even

Reply:
maybe as part of a different PR, we need a good name through